tests/integration/logging/Gopkg.toml

```diff
@@ -22,4 +22,8 @@
 
 required = [
   "golang.org/x/lint/golint"
-]
\ No newline at end of file
+]
+
+[[constraint]]
+  name = "github.com/kyma-project/kyma"
+  version = "1.10.0"
```

Comment:
I am not sure if this is correct. Shouldn't it be master branch? Or are we going to manually bump this on every release?

Reply:
As suggested by @hisarbalik, it is better to make it point to a stable version. Because if the authentication package is broken in the master branch, then this test will break and all PRs to the kyma repo will be blocked